lib/src/connector/connection.cc

```diff
@@ -85,10 +85,12 @@ Connection::Connection(std::vector<std::string> broker_ws_uris,
           consecutive_pong_timeouts_ { 0 },
           endpoint_ { new WS_Client_Type() }
 {
+    // Disable websocket logging until PE-33165 is resolved.
+    setWebSocketLogLevel(leatherman::logging::log_level::none);
+    setWebSocketLogStream(nullptr);
```

Comment:
For clarity, `setWebSocketLogStream` checks whether it's been given `nullptr` and just does nothing in that case. We could just delete this line, but I think it's helpful to have it explicit and make it clear which code needs to change to re-enable the logging.